### PR TITLE
Allow bot to ban users outside server

### PR DESCRIPTION
**Pull request by davidgan1218**
## Summary of Changes
Allows bot to ban users outside of server even if they have never joined server, previously bot could only ban users already in server.

Note: Ban Message is only sent to banned user if they are already in the server because on Discord direct messages can only be sent to users who are in mutual servers.

### CODE REVIEW

File: src/commandDetails/admin/ban.ts [hunk 1]
```diff
@@ -31,11 +31,9 @@ const banExecuteCommand: SapphireMessageExecuteType = async (client, messageFrom
...
     const days = <number>args['days'];
-    // Get the GuildMember object corresponding to the user in the guild
-    // This is needed because we can only ban GuildMembers, not Users
+    // get Guild object corresponding to server
     const guild = await client.guilds.fetch(vars.TARGET_GUILD_ID);
-    const memberInGuild = await guild.members.fetch({ user });
-    if (await banUser(memberInGuild, reason, days)) {
+    if (await banUser(guild, user, reason, days)) {
       return `Successfully banned user ${user.tag} (id: ${user.id}) ${
         days ? `and deleted their messages in the past ${days} days ` : ``
```
Comment (Fan-Yang-284) [inline src/commandDetails/admin/ban.ts:38]: Here, just update `days` to `${pluralize('day', days)}` like in `src/components/admin.ts/makeBanMessage`

**Review by Fan-Yang-284 — CHANGES_REQUESTED** [review]
Just before merging can you push another commit for a small tweak that we missed before this but might as well fix now - otherwise LGTM!

File: src/components/admin.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ const MOD_USER_ID_FOR_BAN_APPEAL: string = vars.MOD_USER_ID_FOR_BAN_APPEAL;
 const makeBanMessage = (reason: string, days?: number): string =>
   `
 Uh oh, you have been banned from the UW Computer Science Club server ${
-    days && `and your messages in the past ${days} ${pluralize('day', days)} have been deleted`
+    days && `and your messages in the past ${days} ${pluralize('day', days)} have been deleted `
```
Comment (Fan-Yang-284) [inline src/components/admin.ts:12]: Edit: It looks like this won't work as intended if `days` is undefined, so just revert it back to the ternary operator

**Review by probro27 — DISMISSED** [review]
@davidgan1218 LGTM! Please do make the changes that Fan mentioned and there are some merge conflicts so please resolve them.

**Comment by Fan-Yang-284** [comment]
There still seem to be a few linter errors (pictured below) so just fix those and it'll be good!

image

**Review by Fan-Yang-284 — APPROVED** [review]

**Pull request merged**